lib/mumuki/classroom/sinatra/massive.rb

```diff
@@ -190,7 +190,7 @@ def guide_progress_at_page(per_page)
     def as_guide_progress_response(guide_progress)
       {
         student: guide_progress.student.uid,
-        guide: guide_progress.guide.slug,
+        guide: guide_progress.guide[:slug],
```

Comment:
Verify if it is still needed. Should delete it or fix it (new guide_progresses won't work)